### PR TITLE
Async sample

### PR DESCRIPTION
**Pull request by ozbenoved**

### CODE REVIEW

File: SDKSamples/app/build.gradle [hunk 1]
```diff
@@ -45,23 +45,17 @@ android {
...
-    implementation("com.nanorep.devconversation:chatintegration:3.5.1.+") { changing = true }
-    implementation("com.nanorep.devconversation:ui:3.5.1.+") { changing = true }
-    implementation("com.nanorep.devcore:accessibility:3.5.0.+") { changing = true }*/
-
-    implementation "com.bold360ai-sdk.core:sdkcore:3.5.1"
-    implementation "com.bold360ai-sdk.conversation:engine:3.5.1"
-    implementation "com.bold360ai-sdk.conversation:chatintegration:3.5.1"
-    implementation "com.bold360ai-sdk.conversation:ui:3.5.1"
-    implementation "com.bold360ai-sdk.core:accessibility:3.5.0"
+    implementation "com.nanorep.devcore:sdkcore:3.6.0.rc2"
```
Comment (tehilaR2) [inline SDKSamples/app/build.gradle]: ??? latest version is rc1

File: SDKSamples/app/build.gradle [hunk 2]
```diff
@@ -45,23 +45,17 @@ android {
...
+    implementation "com.nanorep.devconversation:engine:3.6.0.rc2"
+    implementation "com.nanorep.devconversation:chatintegration:3.6.0.rc2"
```
Comment (tehilaR2) [inline SDKSamples/app/build.gradle]: ??? latest version is rc1

File: SDKSamples/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name">SDK Samples</string>
     <string name="chat_with_agent">Chat with agent</string>
+    <string name="async_chat_with_bold">Async chat with Bold</string>
```
[the remark this answers is not in the document]
Comment (tehilaR2) [reply SDKSamples/app/src/main/res/values/strings.xml]: also, please change the icons to the ones sent in slack

File: SDKSamples/app/build.gradle (hunk 1 above)
Comment (ozbenoved) [reply SDKSamples/app/build.gradle]: ok changed to rc1

File: SDKSamples/app/build.gradle (hunk 2 above)
Comment (ozbenoved) [reply SDKSamples/app/build.gradle]: ok changed to rc1

File: SDKSamples/app/build.gradle [hunk 4]
```diff
@@ -45,23 +45,17 @@ android {
...
+    implementation "com.nanorep.devconversation:ui:3.6.0.rc2"
+    implementation "com.nanorep.devcore:accessibility:3.6.0.rc2"
```
[the remark this answers is not in the document]
Comment (ozbenoved) [reply SDKSamples/app/build.gradle]: ok changed to rc1

File: SDKSamples/app/src/main/res/values/strings.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ozbenoved) [reply SDKSamples/app/src/main/res/values/strings.xml]: ok

File: SDKSamples/app/build.gradle (hunk 1 above)
Comment (ozbenoved) [reply SDKSamples/app/build.gradle]: I think everything needs to be aligned to rc2

**Pull request merged**